docs/source/reference/modding/settings/input.rst

```diff
@@ -134,3 +134,76 @@ This setting has no effect on the camera rotation speed, which is controlled by
 camera sensitivity setting.
 
 This setting can only be configured by editing the settings configuration file.
+
+enable gyroscope
+----------------
+
+:Type:		boolean
+:Range:		True/False
+:Default:	False
+
+Enable support of camera rotation via gyroscope, supported by SDL2.
+Built-in gyroscopes on mobile devices are known to work.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro horizontal axis
+--------------------
+
+:Type:      string
+:Range:     x, y, z, -x, -y, -z
+:Default:   -x
+
+Maps an axis of gyroscope to the horizontal in-game axis. Minus sign inverses left and right directions.
+A setting is for the landscape mode, for other modes the game corrects data automatically.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro vertical axis
+------------------
+
+:Type:      string
+:Range:     x, y, z, -x, -y, -z
+:Default:   y
+
+Maps an axis of gyroscope to the vertical in-game axis. Minus sign inverses up and down directions.
+A setting is for the landscape mode, for other modes the game corrects data automatically.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro input threshold
+--------------------
+
+:Type:		floating point
+:Range:		> 0
+:Default:	0.01
+
+This setting allows to filter a noise (very small values) of rotation speed.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro horizontal sensitivity
+---------------------------
+
+:Type: float
+:Range: >0
+:Default: 1.0
+
+This setting controls the overall gyroscope horizontal sensitivity.
+The default sensitivity is 1.0, with smaller values requiring more device rotation,
+and larger values requiring less.
+
+This setting can only be configured by editing the settings configuration file.
+
+gyro vertical sensitivity
+-------------------------
+
+:Type: float
+:Range: >0
+:Default: 1.0
+
+This setting controls the overall gyroscope vertical sensitivity.
+The default sensitivity is 1.0, with smaller values requiring more device rotation,
+and larger values requiring less.
```

Comment:
The smaller this sensitivity is, the less visible effect the device rotation will have on the vertical camera rotation, and vice versa.